packages/new-polymath-issuer/src/pages/DividendsWizard/Step-2/CsvModal.tsx

```diff
@@ -130,7 +129,12 @@ export const CsvModal: FC<Props> = ({
               },
               {
                 name: csvTaxWithholdingKey,
-                validators: [validators.isNotEmpty],
+                validators: [
+                  validators.isNotEmpty,
+                  validators.isNumber,
+                  validators.isLowerThan(99),
```

Comment:
Shouldn't this be `lowerThanOrEqual`?

Reply:
True, I was messing around with those validations and forgot to change that one